octobot/community/graphql_requests.py

```diff
@@ -40,6 +40,10 @@
         _id
         profitability
     }
+    stats {
+        _id
+        profitability
+    }
```

Comment:
?

Reply:
it's from the backtester branch, I'm removing it